### DISCUSSION TITLE
Remove GuzzleHttp in Prestashop 9

### DISCUSSION THREAD

**Discussion by FabienPapet** [PrestaShop/PrestaShop#30841] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

Guzzle is used only once in the core, and that usage can be replaced by Symfony HttpClient.

### Describe the solution you'd like

`composer remove guzzlehttp/guzzle`

### Do you plan to work on this feature?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by eternoendless** [comment]
This will require a new [client for Circuit breaker](https://github.com/PrestaShop/circuit-breaker/tree/develop/src/Client)
